tezos/messages/src/protocol/proto_012/operation.rs

```diff
@@ -3,69 +3,56 @@
 
 //! Operation contents. This is the contents of the opaque field [super::operation::Operation::data].
 //!
-//! See https://tezos.gitlab.io/protocols/012_ithaca.html
+//! Comparing to the previous protocol, new operation [`RegisterGlobalConstant`] has been added.
+//! See http://tezos.gitlab.io/protocols/011_hangzhou.html#global-constants.
```

Comment:
@akoptelov shouldn't this be in `proto_011/operation.rs` instead?

Reply:
Sure, forgot to update the comment.